lib/kb_ReadsUtilities/kb_ReadsUtilitiesImpl.py

```diff
@@ -81,6 +79,46 @@ def log(self, target, message):
         print(message)
         sys.stdout.flush()
 
+    def add_id_to_plus_line(self,readsfile):
```

Comment:
guess not.  I had it in for a time because I thought it might be the source of a bug I was fixing, but then it turned out not to matter.  I want to leave it available in case we decide to standardize to that format later and activate it.